<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Navigation</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QGraphicsView" name="map_view"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Zoom:</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QSlider" name="zoom_slider">
                 <property name="minimum">
                  <number>100</number>
                 </property>
                 <property name="maximum">
                  <number>500</number>
                 </property>
                 <property name="pageStep">
                  <number>20</number>
                 </property>
                 <property name="value">
                  <number>300</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Destination:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="destination_label">
             <property name="maximumSize">
              <size>
               <width>349</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>2nd floor conference room, room 2999</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="set_destination_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Set Destination</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="x_pos_label">
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="y_pos_label">
             <property name="text">
              <string>value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="update_location_button">
           <property name="text">
            <string>&amp;Update My Location</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="quit_button">
           <property name="text">
            <string>&amp;Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;People and Offices</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>&amp;Labs and Places</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set_destination_button</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
